app/views/posts/_archive.erb

```diff
@@ -0,0 +1,7 @@
+<hr class="small" id="archive" />
```

Comment:
Is this file supposed to be just `.erb` rather than `.html.erb`?

Reply:
No, that's an error on my part.